daml-lf/transaction/src/test/scala/com/digitalasset/daml/lf/transaction/TransactionVersionSpec.scala

```diff
@@ -16,27 +16,33 @@ import scala.collection.immutable.HashMap
 class TransactionVersionSpec extends WordSpec with Matchers {
   import TransactionVersionSpec._
 
+  def fixVersion(v: TransactionVersion) =
+    VersionTimeline.maxVersion(TransactionVersions.minOutputVersion, v)
```

Comment:
I'm not sure this kind of conditional logic in tests helps. Can we just use the resolved values in the assertions, and change the test names to explain why?

It might even make sense to delete some tests.

Reply:
Add a comments.